### PR TITLE
Adding log configuration options in GUI

### PR DESCRIPTION
**Pull request by rupal-bq**
*Issue #12*

*Description of changes:*
-Adding log configuration options

image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/elasticodbc/dlg_specific.h [hunk 1]
```diff
@@ -178,6 +186,12 @@ struct authmode {
     const char *authtype_str;
 };
 const struct authmode *GetAuthModes();
+
+struct loglevel {
+    int loglevel_id;
+    const char *loglevel_str;
```
Comment (lyndonb-bq) [inline src/elasticodbc/dlg_specific.h]: I dont think this str member is being used - you can probably remove this and remove the LOGTYPE_* defines usage.

File: src/elasticodbc/dlg_wingui.c [hunk 2]
```diff
@@ -174,25 +185,74 @@ LRESULT CALLBACK advancedOptionsProc(HWND hdlg, UINT wMsg, WPARAM wParam,
 }
 
 LRESULT logOptionsProc(HWND hdlg, UINT wMsg, WPARAM wParam, LPARAM lParam) {
-    LPSETUPDLG lpsetupdlg;
     ConnInfo *ci;
```
Comment (lyndonb-bq) [inline src/elasticodbc/dlg_wingui.c]: Move initialization into cases where they are used.

File: src/elasticodbc/dlg_wingui.c [hunk 3]
```diff
@@ -134,36 +155,26 @@ static void getDriversDefaultsOfCi(const ConnInfo *ci, GLOBAL_VALUES *glbv) {
 
 LRESULT CALLBACK advancedOptionsProc(HWND hdlg, UINT wMsg, WPARAM wParam,
                                      LPARAM lParam) {
-    LPSETUPDLG lpsetupdlg;
     ConnInfo *ci;
```
Comment (lyndonb-bq) [inline src/elasticodbc/dlg_wingui.c]: Same thing with scoping on this.

File: src/elasticodbc/dlg_specific.h (hunk 1 above)
Comment (rupal-bq) [reply src/elasticodbc/dlg_specific.h]: removed

File: src/elasticodbc/dlg_wingui.c (hunk 2 above)
Comment (rupal-bq) [reply src/elasticodbc/dlg_wingui.c]: moved

**Pull request merged**